### PR TITLE
Make page a whole number

### PR DESCRIPTION
**Pull request by algomaster99**

### CODE REVIEW

File: models/publications/base.py [hunk 1]
```diff
@@ -22,9 +22,9 @@ class BasePublication(models.Model):
 
     year = models.IntegerField()
 
-    pages = models.CharField(
-        max_length=31,
+    pages = models.PositiveIntegerField(
```
Comment (dhruvkb) [inline models/publications/base.py:25]: Afaik, pages are generally given as a range that can take various forms like dashed-ranges ('1-4'), comma-separated values ('1, 2') or even comma-separated combinations of both forms ('1, 3-4').

**Review by dhruvkb — COMMENTED** [review]
That's something I think. @shreyansh23 and @Ajayneethikannan might remember our discussions on the matter more accurately.

File: models/publications/base.py (hunk 1 above)
Comment (algomaster99) [reply models/publications/base.py:25]: Oh, I didn't think of that. This makes sense though.

**Comment by algomaster99** [comment]
https://github.com/IMGIITRoorkee/omniport-service-common-biodata/pull/3#discussion_r450029104 makes complete sense.

**Pull request closed without merging**